js/cobinhood.js

```diff
@@ -529,6 +529,79 @@ module.exports = class cobinhood extends Exchange {
         };
     }
 
+    async fetchDeposits (currency = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        if (typeof currency === 'undefined') {
+            throw new ExchangeError (this.id + ' needs currency for deposit history');
+        }
+        let curr = this.currency (currency);
+        let request = {
+            'limit': limit,
+            'currency': curr['id'],
+        };
+        let response = await this.privateGetWalletDeposits (this.extend (request, params));
+        return this.parseTransactions (response['result']['deposits'], 'deposit');
+    }
+
+    async fetchWithdrawals (currency = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        if (typeof currency === 'undefined') {
+            throw new ExchangeError (this.id + ' needs currency for deposit history');
+        }
+        let curr = this.currency (currency);
+        let request = {
+            'limit': limit,
+            'currency': curr['id'],
+        };
+        let response = await this.privateGetWalletWithdrawals (this.extend (request, params));
+        return this.parseTransactions (response['result']['withdrawals'], 'withdraw');
+    }
+
+    parseTransactionStatus (status) {
+        let statuses = {
+            'tx_pending_two_factor_auth': 'pending',
+            'tx_pending_email_auth': 'pending',
+            'tx_pending_approval': 'pending',
+            'tx_approved': 'pending',
+            'tx_processing': 'pending',
+            'tx_pending': 'pending',
+            'tx_sent': 'pending',
+            'tx_cancelled': 'canceled',
+            'tx_timeout': 'error',
+            'tx_invalid': 'error',
+            'tx_rejected': 'error',
+            'tx_confirmed': 'ok',
+        };
+        return (status in statuses) ? statuses[status] : status.toLowerCase ();
+    }
+
+    parseTransaction (transaction, side = undefined) {
+        let timestamp = this.safeInteger (transaction, 'created_at');
+        let datetime = undefined;
+        if (typeof timestamp !== 'undefined') {
+            datetime = this.iso8601 (timestamp);
+        }
+        let currency = transaction['currency'];
+        if (currency in this.currencies_by_id)
+            currency = this.currencies_by_id[currency]['code'];
+        return {
+            'info': transaction,
+            'id': this.safeString (transaction, 'withdrawal_id'),
+            'txid': this.safeString (transaction, 'txhash'),
+            'timestamp': timestamp,
+            'datetime': datetime,
+            'currency': currency,
+            'status': this.parseTransactionStatus (transaction['status']),
+            'side': side, // direction of the transaction, ('deposit' | 'withdraw')
+            'price': undefined,
```

Comment:
Though it is in the sample data structure, I think that `price` should be removed from the common data format for transactions. I'm awaiting comment on #1896

Reply:
@lfern it looks like `price` should be removed. See discussion [here](https://github.com/ccxt/ccxt/issues/1896#issuecomment-391457166)

Reply:
I submitted a PR fixing the price and other (tested) fixes but @lfern closed it (letting me a bit puzzled...)... so how should I proceed exactly to share my contribution? thx for letting me know...

Reply:
@lfern can you plz accept the edits submitted by @firepol ? ) 

@firepol if @lfern does not accept your edits, then you can either submit a standalone PR (I will merge the two), or I will cherry-pick your edits from the closed one.

Reply:
@kroitor ok I just submitted a standalone PR that basically includes this PR + my fixes. I hope it works like that, cheers

Reply:
@firepol thx! I'm going to merge them this week, as promised.